lesson_4/.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "python.pythonPath": "/home/crashbyte/.pyenv/shims/python"
```

Comment:
fisierele de commit nu le adaugam pe git. O varianta este sa adaugi sau sa modifici fisierul `.gitignore` si in interior sa scrii `.vscode`. Dupa ce ai facut asta nu o sa iti mai apara in git nicio schimbare facuta in `.vscode`